crates/pallet-feeds/src/lib.rs

```diff
@@ -125,6 +140,7 @@ mod pallet {
             feed_id: FeedId,
             object: Object,
             metadata: ObjectMetadata,
+            proof: Proof,
```

Comment:
`proof: Option<Proof>` would be more readable than having `Option` in type alias.